### PR TITLE
Add contract deployment example

### PR DESCRIPTION
**Pull request by jpcenteno**

### CODE REVIEW

**Comment by fkrause98** [comment]
Maybe out of the scope of this PR but, maybe we should add a CI step to check this and future examples compile 
with: `cargo build --examples`, wdy @jpcenteno?

File: examples/contract-deployment/main.rs [hunk 1]
```diff
@@ -0,0 +1,60 @@
+use ethers::abi::Abi;
+use std::str::FromStr;
+use zksync_web3_rs::prelude::abigen;
+use zksync_web3_rs::providers::{Middleware, Provider};
+use zksync_web3_rs::signers::{LocalWallet, Signer};
+use zksync_web3_rs::zks_wallet::DeployRequest;
+use zksync_web3_rs::ZKSWallet;
+
+// This is the default url for a local `era-test-node` instance.
+static ERA_PROVIDER_URL: &str = "http://127.0.0.1:8011";
```
Comment (fkrause98) [inline examples/contract-deployment/main.rs:10]: Is 8011 really the default port? On the readme it says 3050, on my machine it's running on the 3050, this makes the example fail.

**Comment by fkrause98** [comment]
Great work, left some comments.

File: examples/contract-deployment/main.rs (hunk 1 above)
Comment (jpcenteno) [reply examples/contract-deployment/main.rs:10]: Nope, check the following comment. For this example I chose to use the brand-new [era-test-node](https://github.com/matter-labs/era-test-node) which runs faster than the full [local-setup](https://github.com/matter-labs/local-setup) network and it is much easier to run.

```
This is the default url for a local `era-test-node` instance`
```

Comment (jpcenteno) [reply examples/contract-deployment/main.rs:10]: Also, this was manually tested, so I'm certain it works (on my machine, LOL)

**Comment by fkrause98** [comment]
Tested again and got this output:
```
Contract address: 0xb76ed02dea1ba444609602be5d587c4bffd67153
greet: Hey
setGreeting transaction hash 0x272d457c29e13b03f0e2ae7eaa3965484bc6d4ef4892680ba1bc9d61fafd8b69
greet: Hello
```
Great job @jpcenteno ! 😄

**Review by fkrause98 — APPROVED** [review]

**Pull request merged**